tempesta_fw/t/unit/user_space/percentiles.c

```diff
@@ -264,7 +264,7 @@ tfw_stat_adjust(TfwPcntRanges *rng, int r)
 		goto out;
 
 	printf("  -- range %d has outlier %lu (avg=%lu total=%lu) at"
-	       " bucket %lu\n", r, max, sum / cnt, cnt, i_max);
+	       " bucket %lu\n", r, max, sum / cnt, sum, i_max);
```

Comment:
(not for this change, but for the test) There is still no bad test cases? Or did you fix the algorithm by the fix below?